edu-services/gradebook-service/api/src/java/org/sakaiproject/service/gradebook/shared/GradebookService.java

```diff
@@ -683,16 +683,15 @@ public String getAssignmentScoreString(String gradebookUid, Long assignmentId, S
 	
 	/**
 	 * Get student's assignment's score as string.
-	 * This is provided for backward compatibility only.
+	 * This method will attempt to lookup the Long value if provided as a String
+	 * You should use {@link #getAssignmentScoreString(String, Long, String)}
 	 * 
 	 * @param gradebookUid
 	 * @param assignmentName
 	 * @param studentUid
 	 * @return String of score
 	 * 
-	 * @deprecated See {@link #getAssignmentScoreString(String, Long, String)}
 	 */
-	@Deprecated
 	public String getAssignmentScoreString(String gradebookUid, String assignmentName, String studentUid)
```

Comment:
Yeah, I agree. I think I'll just create a new method with more obvious behavior that will do what I'm looking for for forums. (Allow for either strings or numeric values). This seems like the only solution besides a conversion code (which while possible could be difficult but seems like it would require code based scripts) or putting this logic in the tool . . .